pkgs/servers/nas/default.nix

```diff
@@ -0,0 +1,37 @@
+{ stdenv, fetchurl, imake, bison, flex_2_6_1, gccmakedep, xorg }:
+
+let
+  pname = "nas";
+  version = "1.9.4";
+in stdenv.mkDerivation {
+  name = "${pname}-${version}";
+
+  src = fetchurl {
+    url = "mirror://sourceforge/${pname}/${pname}-${version}.src.tar.gz";
+    sha256 = "17dk0ckm6mp1ajc0cd6bwyi638ynw2f6bhbn7gynrs0wfmiyldng";
+  };
+
+  nativeBuildInputs = [ imake bison flex_2_6_1 gccmakedep ];
+
+  buildInputs = with xorg; [ xproto libXau libXt libXext libXaw libXpm ];
```

Comment:
`callPackage` is able to fill in parameters from the `xorg` attrset, so there's no need to pass the whole thing.